src/std/net/json-rpc.ss

```diff
@@ -9,18 +12,23 @@
   :gerbil/gambit/exceptions
   :std/error
   :std/format
+  :std/net/httpd
   :std/net/request
   :std/sugar
   :std/text/json)
 
-(defclass (json-rpc-error jsonable Exception)
-  (code    ;; SInt16
-   message ;; String
-   data)   ;; (Maybe Bytes)
+(defstruct (json-rpc-error <error>) ()
```

Comment:
why not a class extending `Exception`?

Reply:
Because right now, `Exception` isn't really working. But tell me the right thing you want me to do and I'll do it.

Reply:
Let me handle it an a follow up pr